### PR TITLE
implement service page

### PR DESCRIPTION
**Pull request by koya-tech**
# Description

Implemented an service page.
Already responsive to mobile version.

## Type of change

- [x] New feature (non-breaking change which adds functionality)
# Evidence
Screenshot 2024-11-14 225442
Screenshot 2024-11-14 225502-2

### CODE REVIEW

File: scss/partials/service.scss [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+    color: variables.$text-darkbrown;
+}
+
+.services_separator {
+    width: 12.5rem;
+    margin: 0 auto;
+    border-bottom: 2px solid variables.$line-pink;
+}
+
+.services_subtitle {
```
Comment (hello-chloe-seoyeong) [inline scss/partials/service.scss:22]: This part font-family needs to change. from default to sans-serif style. Maybe we need to set and import for second font in variables.scss? Let me know your opinion.

Comment (koya-tech) [reply scss/partials/service.scss:22]: Thank you for your comment.
I found the same refactor point in about page.
I create this issue
https://github.com/Cornerstone-CICCC/web-1-final-project-team3-final/issues/10

So, I will modify code after merging.

**Review by MajitoBrinkitos — APPROVED** [review]

File: scss/partials/service.scss [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+            background-color: variables.$background-light-pink;
+        }
+    }
+
+    .service_business_hour {
+        display: flex;
+        background-color: variables.$background-light-pink;
+
+        .service_business_huor_text {
+            font-family: 'Nunito';
```
Comment (hello-chloe-seoyeong) [inline scss/partials/service.scss:100]: Could you add font-weight to lighter?

File: scss/partials/service.scss [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+.services_subtitle {
+    text-align: center;
+    font-size: 1.125rem;
+    color: variables.$text-black;
+    padding: 2rem;
+}
+
+.services_list {
+    background-color: variables.$background-light-pink;
+    padding: 5rem 13rem;
```
Comment (hello-chloe-seoyeong) [inline scss/partials/service.scss]: If you use max-width, it could be better when responsive.

File: scss/partials/service.scss (hunk 2 above)
Comment (koya-tech) [reply scss/partials/service.scss:100]: I use font weight 300
thanks

File: scss/partials/service.scss (hunk 3 above)
Comment (koya-tech) [reply scss/partials/service.scss]: As I talked with you, I changed code by removing height

**Review by hello-chloe-seoyeong — APPROVED** [review]

**Pull request merged**